src/home_robot/home_robot/motion/ik_solver_base.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Tuple
+
+import numpy as np
+
+
+class IKSolverBase(object):
+    """
+    Base class for all IK solvers.
+    """
+
```

Comment:
Add an `__init__` interface, since all IK solvers should require a URDF file path, ee_link_name, and list of controllable joints? Can also have a field of optional argument that can be used to configure different physics backends.

Reply:
Unfortunately today is my last day at Meta and I need to leave like...now, haha. So I'm creating an issue from this and the tests issue. I'll fix them -- probably Monday. Just trying to get this in to unblock Priyam.